### ISSUE TITLE
Consider switching to bikeshed for a11y reasons

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wcag3#299]
respec, unfortunately has some accessibility issues. Particularly the jump to table of content arrows are links without an accessible name. This problem could be resolved by switching to Bikeshed.

Another problem is actually in using GitHub itself. GitHub does not line wrap HTML files in a pull request in the "unified" view. Arguably this is the right choice for programming languages, but for content, this seems like it would count as a violation of 1.4.10 Reflow. You have to put the content in the "split" view if you want to avoid two-dimensional scrolling. The problem there however is that rendering two files side by side on a significantly zoomed in screen doesn't fit properly either, which is also a violation of 1.4.10 Reflow.

The solve is to use markdown, which GitHub knows can be wrapped safely, and so in unified view, it does line wrap. Because Bikeshed uses markdown instead of HTML, using bikeshed for WCAG 3.0 would solve both problems. Drawback is obviously that Bikeshed has a build step, but this is fairly easy to implement, and is just a one-time task.

**Comment by iadawn** [comment]
The Astro build process that is used to build WCAG 3 works with markdown files. This addresses the issues with how Github presents code. 

The accessible name for table of content links seems to have been addressed in updates to respec. If it still persists then we can get that addressed in respec.

Closing as effectively addressed.
